### ISSUE TITLE
Collada not working with exports from 3ds max

### ISSUE THREAD

**Issue by Crisium** [mrdoob/three.js#7669]
I have the collada pump example working fine. I'm now trying to export an animated dice with 1 texture but in the threejs I only see the shape of the dice with no texture. The animation works fine.

Also the lighting on the dice looks wrong.

I can load the collada file in other program with no problems.

I'm using this version of threejs: https://cdnjs.cloudflare.com/ajax/libs/three.js/r72/three.js

anyone?

ADDED: I have create an example with a cube and one texture. In this example you can see that the texture is not on the cube: http://sigmaphp.dk/threejscollada.zip

**Comment by WestLangley** [comment]
Please rephrase your post as a bug report, reference the example in the zip file, and test against the dev branch first.

Please see "How to report a bug" in the [guidelines](https://github.com/mrdoob/three.js/blob/master/CONTRIBUTING.md).

**Comment by Crisium** [comment]
1. the zip is the example
2. the zip is the latest dev branch.
3. rephrase, please see the "ADDED" part.

this is not rocket science. the example contains a dae that does not work and an example showing that it does not work.

The initial report contains all the information you need.

**Comment by WestLangley** [comment]
> this is not rocket science. the example contains a dae that does not work and an example showing that it does not work.

Actually, I was looking out for you. I did not want your post to be considered a help request and closed. That is why I asked you to rephrase and, as requested in the guidelines, test against the dev branch.

**Comment by GGAlanSmithee** [comment]
Also @Crisium, the tone of your response is not very nice. Try to be nicer when addressing your peers and I'm sure the respect will be returned.

**Comment by mrdoob** [comment]
Seems like the collada file exported from 3ds max doesn't follow the spec (<texture> is supposed to point to a sampler2d, instead it points directly to the texture).

Do you mind trying with the [Open Collada plug-in for 3ds max](http://opencollada.org/)?

**Comment by Crisium** [comment]
@mrdoob I've have load the collada model exported from 3ds max into other collada viewers and they have no problems. I'm thinking it might be a collada version problem, were the newer collada format expects the data as you describe (note: I have not read up on the collada format yet). We have tried the Open Collada on our project which has the same problems, but I have not tried it on the simple cube example I provided here (I'll see if I can test it with Open Collada).

@GGAlanSmithee sorry that you think the tone is not very nice. I was just stating the facts of his arguments to the actual information I provided. I just see too many trolls writting something like "google it", "read your manual" or something like that and this is very rude. I used a lot of time to create a simple example without any animation or anything complex to show the validity of problem. 

@WestLangley thank you for trying to help, but as I can see it the initial comment contains everything that the guideline requires. You might point to "very detailed" description, but I think making a very detailed description would do more harm than good. The problem should be clear and to the point which I believe I have accomplished. Making a long detailed report can produce information making it uncertain what actually was wrong. On that note, there are more problems with the collada loader but I have not included it here because it can cause more confusing to the inital problem at hand.

**Comment by WestLangley** [comment]
@Crisium said

>but as I can see it the initial comment contains everything that the guideline requires.

The demo in your zip file is using r.73. The dev branch is r.74dev. That is why I asked you to test against the dev branch. The request that you test using the dev branch is also made in the guidelines.

**Comment by mrdoob** [comment]
> @mrdoob I've have load the collada model exported from 3ds max into other collada viewers and they have no problems.

Out of curiosity, what viewers are those? Preview.app in OSX does not show the texture either.

> I'm thinking it might be a collada version problem, were the newer collada format expects the data as you describe (note: I have not read up on the collada format yet)

Your file uses version 1.4. Here's the spec: https://www.khronos.org/files/collada_spec_1_4.pdf

**Comment by Crisium** [comment]
@WestLangley I have now downloaded the dev branch instead of the cbn and tested it and the result is still the same problem. There actually was no need to use the dev branch since the collada code in the zip file is from the dev branch. Also the problem is not the render, but the collada loader.

@mrdoob The dae file works on are:
Microsoft Developer Studio (embedded viewer) (just drop the .dae file into developer studio).
open3mod: http://www.open3mod.com/

but... I also tried google sketcher and it reported missing texture.

thanks for the specs I'll take a look at them today.

**Comment by mrdoob** [comment]
Have you tried Open Collada yet?

**Comment by Crisium** [comment]
Yes,.. I have tried with open-collada. Here is a link with the same test but with open collada: https://drive.google.com/file/d/0B88yTIMYNMU0cnE1SDZmdU5wOVU/view?usp=sharing

**Comment by mrdoob** [comment]
`ColladaLoader2` can now parses `smile_opencollada.DAE` properly.

**Comment by Crisium** [comment]
I just tested it with the newest code from githud (downloaded). I can now see the cube but there still is no tectuxe. The cube itself just changes colors every time I run it.

I testing against the ColladaLoader2.

Is there something I'm missing?

**Comment by mrdoob** [comment]
Maybe this... https://github.com/mrdoob/three.js/wiki/How-to-run-things-locally

**Comment by Crisium** [comment]
no.. it's not a same origin problem. the example is running on a local server.

**Comment by mrdoob** [comment]
I'm out of ideas then. Aspect ratio aside... it works fine here.

![screen shot 2015-12-07 at 14 41 53](https://cloud.githubusercontent.com/assets/97088/11637673/b9c67798-9cf0-11e5-8b26-be4cff2f98c5.png)

**Comment by Crisium** [comment]
@mrdoob ,.. I got my example working (smiley face cube) with your fixes. I had to download the development code and not the master. 

I know tried to added the development code to an animated version in our main project. The object is not to be seen and there are no output errors.

Apparently something is not working correctly when animation is involved. I'll try and make a simple example that shows this.
